server/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/TextPreprocessor.java

```diff
@@ -39,4 +39,6 @@
       CopybookConfig copybookConfig,
       Deque<List<Pair<String, String>>> recursiveReplaceStmtStack,
       List<Pair<String, String>> replacingClauses);
+
+  ResultWithErrors<String> codeCleanUp(String documentUri, String cobolCode);
```

Comment:
Optional - please, add javadocs to these methods to show the difference of their purpose